content/millennium/r4/financial/support/coverage.md

```diff
@@ -265,7 +265,7 @@ _Implementation Notes_
 
 * This implementation follows the [JSON Patch](https://tools.ietf.org/html/rfc6902) spec.
 * Only operations on the paths listed below are supported.
-* For Private Coverages, only Encounter-level Coverages may be patched.
+* Patient-level private coverages are supported minus the relationship and subscriber replace operations.
```

Comment:
Need confirmation from Alex , if we need to modify the language or add anything.

Reply:
Let's update this to say "For Private Coverages, both Encounter-level and Patient-level Coverages may be patched, with the caveat of only supporting the /dependent and /period operations on Patient-level Coverages.

Reply:
Updated the note: https://github.com/cerner/fhir.cerner.com/pull/820/files#diff-b606e00be5c937b992ec15dc4884b218073809d10c27279c771f072d522ded27R268

Reply:
Thanks.